app/controllers/user.js

```diff
@@ -31,3 +31,26 @@ exports.signIn = (req, res, next) =>
       }
     })
     .catch(next);
+
+exports.getAllUsers = (req, res, next) => {
+  const limit = req.query.limit || 10;
+  const page = req.query.page || 1;
+  const paginationParams = {
+    limit,
+    page,
+    offset: (page - 1) * limit,
+    orderBy: req.query.orderBy || 'email',
+    order:
+      req.query.order && (req.query.order.toUpperCase() === 'ASC' || req.query.order.toUpperCase() === 'DESC')
```

Comment:
You could build a helper or a function for this, also define ASC and DESC as constants

Reply:
You also could check query param order with express-validator